src/plots/plots.js

```diff
@@ -715,6 +750,42 @@ plots.supplyDataDefaults = function(traceIn, traceIndex, layout) {
     return traceOut;
 };
 
+function supplyTransformDefaults(traceIn, traceOut, layout) {
+    var containerIn = traceIn.transforms || [],
+        containerOut = traceOut.transforms = [];
```

Comment:
just verified.
putting a 
```javascript
    if(!Array.isArray(traceIn.transforms)) {
        return;
    }
```
at the top of this function remedies the issue. Though perhaps this breaks other things. Not sure.